src/perturb.jl

```diff
@@ -29,8 +29,6 @@ function create_perturbation_gates(s, k, B_pert, α, x, L, N_sites, energy_sign,
     gates = ITensor[] 
 
     # define an array of oscillation frequencies (units of ergs) of perturbation
-    # ω_pert = [10^(-28)* energy_sign[i] for i in 1:N_sites]
-    # ω_pert = fill(2π*(10^(-26)), N_sites) 
     ω_pert = asin.([α * sin(k*x[i])* energy_sign[i] for i in 1:N_sites])
```

Comment:
I think the description of the input parameters needs to be adjusted based on the arguments to the function.